src/NServiceBus.AcceptanceTests/Sagas/When_finder_cant_find_saga_instance.cs

```diff
@@ -12,7 +12,7 @@
     [TestFixture]
     public class When_finder_cant_find_saga_instance : NServiceBusAcceptanceTest
     {
-        [Test]
+        [Test, Ignore("Not sure what to do here, since the start message has no corr prop we can't generate the SagaId, perhaps custom finder isn't supported for the devstorage?")]
```

Comment:
Moved the test to the core folder and made it use the InMemory persister for now